infra/base-images/base-clang/Dockerfile

```diff
@@ -18,16 +18,23 @@
 
 FROM gcr.io/oss-fuzz-base/base-image
 
+ARG introspector
+ENV INTROSPECTOR_PATCHES=$introspector
+
 # Install newer cmake.
 ENV CMAKE_VERSION 3.21.1
-RUN apt-get update && apt-get install -y wget sudo && \
+RUN apt-get update && apt-get install -y wget sudo git && \
```

Comment:
I don't really love the placement of this install. This line is meant to install cmake and deletes the packages used to build it., I think instead, you should install git on line 34 and then uninstall it in the same step.
I think we should uninstall it there because it will just add bloat to the image size since in checkout_build_install_llvm.sh we install git and then uninstall it.

Reply:
addressed in  #7199